tests/playbooks/tests_eth_dns_support.yml

```diff
@@ -10,7 +10,7 @@
 
 
   tasks:
-    - name: "set type={{ type }} and interface={{ interface }}"
+    - name: "set type=veth and interface={{ interface }}"
```

Comment:
What about `set interface and type to {{ interface }} {{ type }}` this should avoid triggering the error and it will continue using the variable. What do you think?

Reply:
Also this applies to others similar

Reply:
Not sure.  In general, Ansible considers using `{{ ... }}` in the `name:` of a task - at all, anywhere - to be a bad idea.  For one, it makes "grepping" the source code for errors from the logs much harder.

Reply:
@richm , so just avoid using `{{...}}` in the name ?  And replace the variable with the real value?

Reply:
>I think for the work you are doing, in order not to create more work, if you have already fixed name: values to have a single {{ ... }} at the end of the string, keep it. Do not change it.

>For name: values that still have one or more {{ ... }} in them not at the end of the string, consider rewriting the name: value to get rid of all {{ ... }} in the name: value.

If so, then this PR is ready for review.